src/RoomInvite.js

```diff
@@ -23,6 +23,7 @@ import Modal from './Modal';
 import * as sdk from './';
 import { _t } from './languageHandler';
 import {KIND_DM, KIND_INVITE} from "./components/views/dialogs/InviteDialog";
+import PrototypeCommunityInviteDialog from "./components/views/dialogs/PrototypeCommunityInviteDialog";
```

Comment:
Hmm, the store in the previous PR was named `CommunityPrototype` instead of `PrototypeCommunity`, can we use the same prefix consistently?